rasa_core/channels/facebook.py

```diff
@@ -50,6 +57,9 @@ def message(self, message: Dict[Text, Any]) -> None:
         elif self._is_audio_message(message):
             attachment = message['message']['attachments'][0]
             text = attachment['payload']['url']
+        elif self._is_attachment_message(message):
+            attachment = message['message']['attachments'][0]
```

Comment:
what do you think of the idea to support multiple attachments? We could split the different urls by a newline in the code. E.g.

```python
attachments = []
for attachment in message['message'].get('attachments', []):
    url = attachment.get('payload', {}).get('url')
    attachments.append(url)

text = '\n'.join(attachments)
```

Reply:
As far as I remember, I have tested it on facebook messenger(weeks ago BTY). I typed some texts and added 2 attachments into one message, then sent it out.  The backend actually got 3 messages separately. Each message only has one type of content. One for text and other two for attachments.

Reply:
👍 Cool, that you tried it out. Is it the same if you send multiple images?